### ISSUE TITLE
Values for a graphic representation

### ISSUE THREAD

**Issue by marcelolustosa** [RichardRed0x/crypto-governance-research#11]
I need to apply values to your table in order to draw a chart. Have I guessed them properly in the image below? I've set 0 for (not applicable) and 5 for (direct), being 3 for (delegated).

![staking-3-dimensions-mockup](https://user-images.githubusercontent.com/39598223/58909311-27846200-86e9-11e9-90b0-4fe90d85e85f.png)

**Comment by RichardRed0x** [comment]
The scale I am using is 0-1 or 0-100, with NA values having their own space in the graphic to the left of that. In the data-set [draft](https://github.com/RichardRed0x/crypto-governance-research/blob/staking-overview/stake-based-governance/staking_overview.csv) I have at the moment the scale is 0-1.

**Comment by marcelolustosa** [comment]
Ah great!

**Comment by Dustorf** [comment]
Lustosa-there will likely one simplified chart for my purpose, and perhaps multiple, more complex outputs for the more in-depth piece Richard is starting. As such, I'd love to draw this on a triangular axis for me with zero being the center and 100 being the outside point of the triangle. Each project will be represented by their connected "triangles" within the larger one. LMK if that makes sense

**Comment by marcelolustosa** [comment]
This is the raw chart I'll be working with, generated from Richard's data.

There's a issue in differentiating here the Tezos 'zero' score from EOS 'not applicable', in Funding.

![Capturar](https://user-images.githubusercontent.com/39598223/59707408-f3289f80-91d8-11e9-8539-1820475cb4fa.PNG)

**Comment by Dustorf** [comment]
My first thought is that this is designed to be high level, and that 0 = N/A

**Comment by marcelolustosa** [comment]
nice, I do agree, but wanted to be sure

**Comment by Dustorf** [comment]
Thumbs up Stakey

**Comment by marcelolustosa** [comment]
![dcr--governance](https://user-images.githubusercontent.com/39598223/59717843-355cdb80-91ef-11e9-839b-8bc682cced96.png)

**Comment by RichardRed0x** [comment]
I don't think this format works well for a static image, too much going on and unfamiliar makes it confusing. In my opinion this kind of graphic works better when it's live and dynamic, showing one project's data with the user in control.

Looks like the color legend is wrong?

**Comment by marcelolustosa** [comment]
True, I've switched Dash and Decred by mistake:
![dcr--governance](https://user-images.githubusercontent.com/39598223/59724898-fe43f580-9201-11e9-849a-29333a76031b.png)

This is for Dustorf, idk how he's going to use it. Later on, as he suggested, we can make an interactive version of it. Just need to know where it will be hosted.

**Comment by RichardRed0x** [comment]
Fair enough. If you're making it manually I should flag that the numbers will need to be refined before a published version, I can see a couple of issues already.

**Comment by cryptoleslie** [comment]
Marcel's idea of an interactive version sounds exciting. If we go with the chart in your first comment, maybe add arrows to the lines to indicate going up in value is positive? I think a more refined version of this all-in-one is a good direction for us. Thank you for your work on this.

**Comment by cryptoleslie** [comment]
Ideally, let's get the static model looking better (I see pixelation in the mockup on my end), then go to the interactive. We'll want to have both available.

**Comment by marcelolustosa** [comment]
We can use a JS library to make it interactive if we are willing to host it. For Medium or similar platforms, it can be an animated gif showing each project rating then the full chart.

**Comment by cryptoleslie** [comment]
Yes, I can see this interactive JS version becoming a gif on Twitter, and perhaps also hosted on our website. Is that in line with your idea, Marcelolustosa?

**Comment by RichardRed0x** [comment]
I have updated the values for Dash and Tezos [here](https://github.com/RichardRed0x/crypto-governance-research/blob/staking-overview/stake-based-governance/staking_overview.csv), and added a Dash writeup [here](https://github.com/RichardRed0x/crypto-governance-research/blob/staking-overview/stake-based-governance/stake-based-overview.md).
